GetCertificate/printer.ts

```diff
@@ -109,13 +139,46 @@ export const printUvci = (
     .when(RecoveryCertificate.is, cr => cr.r[0].ci)
     .exhaustive(); // TODO
 
+/**
+ * Check if Vaccination has ended
+ *
+ * @param v the VaccinationEntry to check
+ * @returns true if process ended, false otherwise
+ */
 export const isVaccinationProcessEnded = (v: VaccinationEntry): boolean =>
   v.dn === v.sd;
 
+/**
+ * Format date value based on language
+ *
+ * @param d the date
+ * @param _lang the preferred language
+ * @returns a formatted date
+ */
 export const formatDate = (d: Date, _lang: PreferredLanguage): string =>
   match(_lang)
     .when(
       l => l === PreferredLanguageEnum.en_GB,
       _ => moment(d).format(DATE_FORMAT_EN)
     )
     .otherwise(() => moment(d).format(DATE_FORMAT_ITA));
+
+/**
+ * Format Certificate Issuer based on its value and preferred language
+ *
+ * @param c the issuer certificate value
+ * @param _lang the preferred language
+ * @returns a formatted Certificate Issuer string
+ */
+export const formatCertificateIssuer = (
+  c: string,
+  _lang: PreferredLanguage
```

Comment:
What is the reason for the underscore prefix? Underscore is typically used for unused variables but this is.

Reply:
fixed, thank you!